knimin/templates/pm_normalize.html

```diff
@@ -0,0 +1,125 @@
+{% extends logged_in_index.html %}
+{% block head %}
+{% from json import dumps %}
+<script type="text/javascript">
+
+  function validateForm(e) {
+    var success = true;
+
+    if ($("#upload-select").val() === 'Single file') {
+      $("#single-plate-fp").parents(':first').removeClass('has-error');
+      if($("#single-plate-fp").val() === '') {
+        success = false;
+        $("#single-plate-fp").parents(':first').addClass('has-error');
+      }
+    } else {
+      for (var i = 0; i < 4; i++) {
+        var $inp = $('#plate-' + i + '-fp');
+        $inp.parents(':first').removeClass('has-error');
+        if ($inp.val() === ''){
+          success = false;
+          $inp.parents(':first').addClass('has-error');
+        }
+      }
+    }
+
+    // On submission, the form will open a new tab to start downloading an
+    // echo pick file. We want this page to redirect to the next step on the
+    // workflow. That is accomplished by adding a timeout function to perform
+    // the redirect
+    if (success){
+      setTimeout(function () {
+        window.location.href = "/pm_shotgun_pool/?plate={{plate['id']}}";
+      }, 1000);
```

Comment:
Developer decision :P I just need that after the post this page redirects, so any short period of time works for me.